lib/chef/knife/solo_prepare.rb

```diff
@@ -47,6 +47,10 @@ class SoloPrepare < Knife
           name, path = h.split("=")
           Chef::Config[:knife][:hints][name] = path ? JSON.parse(::File.read(path)) : Hash.new  }
 
+     option :lock_file,
```

Comment:
I may have not read the code correctly, but I wanted the command to be available for both `knife solo cook`, `knife solo prepare`, and `knife solo bootstrap`. I had thought prepare would fail without it, maybe that's an oversight.